roles/network/defaults/main.yml

```diff
@@ -91,7 +91,6 @@ systemd_networkd_active: False
 # The values here are default local variables
 gui_wan_iface: "unset"
 gui_static_wan_ip: "unset"
-gui_desired_network_role: Gateway
```

Comment:
I think role defaults have the lowest precedence, below ./vars/default_vars, which is below /etc/iiab/local_vars.yml